src/widget/woverview.cpp

```diff
@@ -1122,7 +1124,7 @@ void WOverview::drawMarks(QPainter* pPainter, const float offset, const float ga
             double markTime = samplePositionToSeconds(markSamples);
             double markTimeRemaining = samplePositionToSeconds(trackSamples - markSamples);
             double markTimeDistance = samplePositionToSeconds(markSamples - currentPositionSamples);
-            QString cuePositionText = mixxx::Duration::formatTime(markTime) + " -" +
+            const QString cuePositionText = mixxx::Duration::formatTime(markTime) + " -" +
```

Comment:
is this missing a space?

Reply:
This is the time remaining until the track end, it's always negative (except at the end)
In this case-0:09
![image](https://github.com/user-attachments/assets/7135c3b7-9a88-4d0a-b1f5-5927c67cea17)

Reply:
Ah, right. of course